src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -780,8 +780,9 @@ public static void mutateMV(ByteBuffer dataKey, Collection<Mutation> mutations,
                 ConsistencyLevel consistencyLevel = ConsistencyLevel.ONE;
 
                 //Since the base -> view replication is 1:1 we only need to store the BL locally
-                final Collection<InetAddressAndPort> batchlogEndpoints = Collections.singleton(FBUtilities.getBroadcastAddressAndPort());
-                BatchlogResponseHandler.BatchlogCleanup cleanup = new BatchlogResponseHandler.BatchlogCleanup(mutations.size(), () -> asyncRemoveFromBatchlog(batchlogEndpoints, batchUUID));
+                ReplicaPlan.ForTokenWrite replicaPlan = ReplicaPlans.forLocalBatchlogWrite();
+                BatchlogResponseHandler.BatchlogCleanup cleanup = new BatchlogResponseHandler.BatchlogCleanup(mutations.size(),
```

Comment:
If we static import the inner class, we can avoid the unnecessary newline (which irks me unreasonably, because it's anyway going over the line length limit)

Reply:
Fixed